pages/common/ts-node.md

```diff
@@ -0,0 +1,28 @@
+# ts-node
+
+> Runs TypeScript code directly, without any compiling.
+> More information at: https://www.npmjs.com/package/ts-node
+
+- Execute a TS File `foobar.ts` without compiling ('node' + 'tsc')
+
+`ts-node {{foobar.ts}}`
+
+- TS-Node Loads tsconfig.json by default. To prevent that
+
+`ts-node --skip-project {{foobar.ts}}`
+
+- Loads TS-Node Help
+
+`ts-node --help`
+
+- Eval TS Code directly on the terminal
+
+`ts-node -e 'console.log("Hello World")'`
```

Comment:
```suggestion
- Evaluate TS code directly on the terminal:

`ts-node -e '{{console.log("Hello World")}}'`
```

Reply:
I'd also like it to be "Evaluate TypeScript code ..."

Reply:
Should we generally replace `TS` with `TypeScript`? The descriptions aren't really long so that wouldn't be a problem.

Reply:
Yes, I think so.  I would have waited to send that comment but I didn't notice that we were using `TS` all over before that... 🤣